opengamedata/opengamedata-website
# FD-36 monthly usage chart updates

Testing instructions:

- Make sure API backend is running
- Check new monthly usage chart dataset and interactions
- Chart defaults to 30 months
- Nav moves from month to month, loading a new set once it you reach the end of a block of 30 months.
- Click on a bar to select that month and view any files for that month
